responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -7,6 +7,7 @@
 from dice_ml import Dice
 import numpy as np
 
+from responsibleai._data_validations import _validate_train_test_categories
```

Comment:
FYI Looks like maybe you forgot to `git add` this file

Reply:
Done.